.github/workflows/integrational-tests.yml

```
@@ -10,8 +10,9 @@ env:
   HAZELCAST_VERSION: 4.2
 
 jobs:
+
   run-tests:
-    name: Run Intergrational tests on GKE
+    name: Run Integrational tests on GKE
```

Although the spelling fix looks right, I don't think "integrational" is a word. I realize the name of this file is "integrational-tests", can also we change it to "integration-tests" while we are at it? 
```suggestion
    name: Run Integration tests on GKE
```

---

changed as suggested.